srcpkgs/r2ghidra-dec/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'r2ghidra-dec'
+_cutterversion=1.11.1
+pkgname=r2ghidra-dec
+version=4.5.0
+revision=1
+archs="x86_64"
+build_style=cmake
+configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DBUILD_CUTTER_PLUGIN=ON -DCUTTER_SOURCE_DIR=../cutter .."
+hostmakedepends="pkg-config cmake git bison flex"
+makedepends="radare2 qt5-devel capstone-devel"
+depends="radare2"
+short_desc="Deep ghidra decompiler integration for radare2 and Cutter"
+maintainer="phire <me@phire.cc>"
+license="LGPL-3.0-only"
+homepage="https://github.com/radareorg/r2ghidra-dec"
+
+do_fetch() {
+	git clone --depth 1 --branch v${version} https://github.com/radareorg/r2ghidra-dec.git ${wrksrc}
+	git clone --depth 1 --branch v${_cutterversion} https://github.com/radareorg/cutter.git ${wrksrc}/cutter
+	cd ${wrksrc}
+	git submodule init
+	git submodule update --recursive
+}
```

Comment:
This isn't a good solution. The build can change if they make updates to the branch. You should fetch all the files, including submodules, by adding a definitive file to `distfiles`. So either a tag or specific commit.

Reply:
Updated